web/src/components/core/ShowLogButton.jsx

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright (c) [2023] SUSE LLC
- *
- * All Rights Reserved.
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms of version 2 of the GNU General Public License as published
- * by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
- * more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, contact SUSE LLC.
- *
- * To contact SUSE LLC about this file by physical or electronic mail, you may
- * find current contact information at www.suse.com.
- */
-
-import React, { useState } from "react";
-import { FileViewer } from "~/components/core";
-import { Icon } from "~/components/layout";
-import { Button } from "@patternfly/react-core";
-import { _ } from "~/i18n";
-
-/**
- * Button for displaying the YaST logs
- * @component
- */
-const ShowLogButton = () => {
-  const [isLogDisplayed, setIsLogDisplayed] = useState(false);
-
-  const onClick = () => setIsLogDisplayed(true);
-  const onClose = () => setIsLogDisplayed(false);
-
-  return (
-    <>
-      <Button
-        variant="link"
-        onClick={onClick}
-        isDisabled={isLogDisplayed}
-        icon={<Icon name="description" size="s" />}
-      >
-        {/* TRANSLATORS: button label */}
-        {_("Show Logs")}
-      </Button>
-
-      { isLogDisplayed &&
-        <FileViewer
```

Comment:
I did not meant to drop the buttons completely (just removing the from the menu). But I am not against dropping that code. In that case, you should remove `FileViewer` too.

Reply:
We were also not sure, that is why a I put it in a separate commit just to revert in case.. :) Will remove it only from the menu by now as it does not hurt and probably we will bring them back soon.